src/appleseed/renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp

```diff
@@ -120,9 +121,9 @@ namespace
             const float font_height = 14.0f * m_scale_factor;
             const Color4f FontColor(srgb_to_linear_rgb(Color3f(0.9f, 0.9f, 0.9f)), 1.0f);   // linear RGB
             const Color4f BackgroundColor(0.0f, 0.0f, 0.0f, 0.95f);                         // linear RGB
-            const Color4f LogoTint(1.0f, 1.0f, 1.0f, 0.8f);                                 // linear RGB
-            const float MarginH = 6.0f;
-            const float MarginV = 4.0f;
+            const Color4f LogoTint(0.85f, 0.85f, 0.85f, 0.8f);                              // linear RGB
```

Comment:
I'm not sure what this is needed. The icons in `resources/logo/` are the canonical ones. 

In the final render with render stamp, the orange should be `#FF5A11` (255, 90, 17) and the gray should be `#C8C8C8` (200, 200, 200).

You should be able to get exactly that without any tinting.

Reply:
Note that these colors are expressed in sRGB.